validphys2/src/validphys/convolution.py

```diff
@@ -112,13 +112,13 @@ def _subtract_abspair(a, b, c, d):
     Parameters:
     -----------
     a: np.ndarray
-        polarized tensor-like object
+        Unpolarized array-like object
     b: np.ndarray
-        polarized tensor-like object
+        Unpolarized array-like object
     c: np.ndarray
-        Unpolarized tensor-like object
+        polarized array-like object
     d: np.ndarray
-        Unpolarized tensor-like object
+        polarized array-like object
```

Comment:
Please, remove the strong references to the polarize (I put a suggestion above as an example) in this and the dosctring above. In principle these functions, even if the target is polarized fits, are generally valid (and for sure, the input itself is not polarized).

(I think there is value on making it clear that the expectation is unp unp pol pol, but a "numpy unpolarized array" might a bit confusing!)